### PR TITLE
Create base mailer with send email ability

### PR DESCRIPTION
**Pull request by dkryvi**

### CODE REVIEW

File: app/services/mailer.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  secure: true,
+  auth: {
+    user: nconf.get('mailer:user'),
+    pass: nconf.get('mailer:pass'),
+  },
+  logger
+});
+
+const defaultEmailOptions = {
+  from: 'EventEmitterService@gmail.com',
```
Comment (akosenko) [inline app/services/mailer.js]: move to config

File: app/config/index.js [hunk 2]
```diff
@@ -16,4 +18,5 @@ nconf
   .file('envConfig', path.join(__dirname, `${env}.json`))
   .defaults(defaults);
 
-module.exports = nconf;
+nconf.set('mailer:user', process.env.EMAIL_USER);
+nconf.set('mailer:pass', process.env.EMAIL_PASS);
```
Comment (akosenko) [inline app/config/index.js:22]: move to config

**Review by akosenko — CHANGES_REQUESTED** [review]

File: app/config/index.js (hunk 2 above)
Comment (DenKrivichanin) [reply app/config/index.js]: [Done](https://github.com/inst-event-emitter/eventemitter-backend/pull/20/commits/f5f1e52333bd5100b40935f350b7a08a372abf79#diff-a18e55398c3a9a4054717c2ccfa889fdR16)

File: app/services/mailer.js (hunk 1 above)
Comment (dkryvi) [reply app/services/mailer.js]: [Done](https://github.com/inst-event-emitter/eventemitter-backend/pull/20/commits/f5f1e52333bd5100b40935f350b7a08a372abf79#diff-a18e55398c3a9a4054717c2ccfa889fdR16)

File: app/config/index.js (hunk 2 above)
Comment (dkryvi) [reply app/config/index.js:22]: I can't make email and pass public. I think it's not secure

Comment (akosenko) [reply app/config/index.js:22]: so, please make corresponding changes in jenkins

**Review by akosenko — APPROVED** [review]

File: app/config/index.js (hunk 2 above)
Comment (dkryvi) [reply app/config/index.js:22]: Okay. I will add these env variables in Jenkins conf

**Comment by akosenko** [comment]
Please, resolve conflicts

**Pull request merged**